File: uri/1131/1131.cpp
#include <cstdio>

int main(){
    int g, i, gv = 0, iv = 0, e = 0, n = 0, r;
    do {
        scanf("%d %d", &i, &g);
        n++;
        if (i == g)
            e++;
        if (i > g)
            iv++;
        else
            gv++;
        printf("Novo grenal (1-sim 2-nao)\n");
    } while (scanf("%d", &r) && r == 1);
    printf("%d grenais\n", n);
    printf("Inter:%d\n", iv);
    printf("Gremio:%d\n", gv);
    printf("Empates:%d\n", e);
    if (iv > gv)
        printf("Inter venceu mais\n");
    else if (gv > iv)
        printf("Gremio venceu mais\n");
    else
        printf("Nao houve vencedor\n");
    return 0;
}

File: solutions/beecrowd/1131/1131.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

n = e = iv = gv = 0

while True:
    i, g = map(int, input().split())
    n += 1

    if i == g:
        e += 1

    if i > g:
        iv += 1
    else:
        gv += 1

    output('Novo grenal (1-sim 2-nao)\n')
    if (r := int(input())) == 2:
        break

output(f'{n} grenais\n')
output(f'Inter:{iv}\n')
output(f'Gremio:{gv}\n')
output(f'Empates:{e}\n')

if iv > gv:
    output('Inter venceu mais\n')
elif gv > iv:
    output('Inter venceu mais\n')
else:
    output('Não houve vencedor\n')
